src/crystal/system/print_error.cr

```diff
@@ -0,0 +1,12 @@
+module Crystal::System
+  # Prints directly to stderr without going through an IO.
+  # This is useful for error messages from components that are required for
+  # IO to work (fibers, scheduler, event_loop).
+  def self.print_error(message)
```

Comment:
I'd add a simple logic for appending `\n` to the *message*, since it's the most common usage case.
Perhaps sth along the lines of `message = message + "\n" unless message.ends_with?('\n')`?

Reply:
This is a very low-level method. I think it is fine without convenient, but ultimately unnecessary logic enhancements.